curriculum/challenges/english/25-front-end-development/lab-leap-year-calculator/66c06fad3475cd92421b9ac2.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 66c06fad3475cd92421b9ac2
+title: Build a Leap Year Calculator
+challengeType: 14
+dashedName: build-a-leap-year-calculator
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. Declare a variable `year` that stores the value of the year to check.
+
+1. Use an `if/ else` statement to check if the year is a leap year.
+
+1. To check if the year is a leap year, fulfill the following conditions in the `if` part of the statement:
```

Comment:
I think we should leave off the `if` statement part because technically this would be a valid solution

```js
const year = 2024;
const isLeapYear = (year % 4 === 0 && year % 100 !== 0) || year % 400 === 0;
console.log(`${year} is ${isLeapYear ? "a leap year" : "not a leap year"}.`);

```